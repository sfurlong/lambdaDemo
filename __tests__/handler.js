const { handler } = require('../src/handler')

test('It should respond with hello world', () => {
  handler({}, {}, (err, response) => {
    expect(err).toBe(null)
    expect(response).toMatchSnapshot()
  })
})
