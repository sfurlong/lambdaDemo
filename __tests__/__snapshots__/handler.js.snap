// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should respond with hello world 1`] = `
Object {
  "body": "{\\"message\\":\\"Hello World\\"}",
  "headers": Object {
    "Content-Type": "application/json",
  },
  "statusCode": 200,
}
`;
